FixedSizeIntegerArray.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Scanner;
+
+public class FixedSizeIntegerArray {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter integer (n): ");
+        int n = scanner.nextInt();
+
+        if (n <= 0 || n > 8) {
+            System.out.println("Maximum size should be 8 and minimum should be 1.");
+            System.exit(1);
+        }
+
+        int[] array = new int[n];
+
+        System.out.print("Enter first number (a): ");
+        int a = scanner.nextInt();
+
+        System.out.print("Enter second number (b): ");
+        int b = scanner.nextInt();
+
+        int smaller, larger;
+        if (a < b) {
+            smaller = a;
+            larger = b;
+        } else {
+            smaller = b;
+            larger = a;
+        }
+
+        int middleIndex = n / 2;
+        if (n > 1) {
+            array[middleIndex - 1] = smaller;
+            if (middleIndex < n) {
```

Comment:
Why is this condition? Is there any probability, that `middleIndex` can more than equal to `n` ?

Reply:
The comment is fixed: the condition  is corrected